docs/content/integrations/airflow.mdx

```diff
@@ -1,18 +1,135 @@
+description: The dagster-airflow package allows you to export Dagster pipelines as Airflow DAGs, as well as to import Airflow DAGs into Dagster pipelines.
 ---
 
 # Using Dagster with Airflow
 
 <CodeReferenceLink filePath="examples/airflow_ingest" />
 
-The [`dagster-airflow`](/\_apidocs/libraries/dagster_airflow) package allows users to run Dagster pipelines on Airflow clusters, and to import Airflow DAGs into Dagster instances.
+The [`dagster-airflow`](/\_apidocs/libraries/dagster_airflow) package allows you to export Dagster pipelines as Airflow DAGs, as well as to import Airflow DAGs into Dagster pipelines.
 
-This is not recommended for new projects. This is designed for users who want to use the Dagster API within an existing Airflow cluster, and for those who want tooling support to migrate away from Airflow. Using Dagster within an Airflow limits its capabilities, as you still have to rely on Airflow's scheduler, and not all Dagster features (e.g. manual retries from Dagit) will work.
+Dagster is a fully-featured orchestrator and does not require a system like Airflow to deploy, execute, or schedule pipelines. The main scenarios for using Dagster with Airflow are:
 
-## Airflow Ingest
+- You have an existing Airflow setup that's too difficult to migrate away from, but you want to use Dagster for local development.
+- You want to migrate from Airflow to Dagster in an incremental fashion.
 
-This example demonstrates how to use <PyObject   module="dagster_airflow"   object="make_dagster_pipeline_from_airflow_dag" /> to compile an Airflow DAG into a Dagster pipeline that can be executed (and explored) the same way as a Dagster-native pipeline.
+## Exporting Dagster pipelines to Airflow
+
+You can compile Dagster pipelines into DAGs that can be understood by Airflow. Each solid in the pipeline becomes and Airflow task. Here's a pipeline defined using Dagster:
+
+```python file=/integrations/airflow/hello_cereal.py
+import csv
+
+import requests
+from dagster import ModeDefinition, fs_io_manager, pipeline, solid
+
+
+@solid
+def download_cereals():
+    response = requests.get("https://docs.dagster.io/assets/cereal.csv")
+    lines = response.text.split("\n")
+    return [row for row in csv.DictReader(lines)]
+
+
+@solid
+def find_sugariest(context, cereals):
+    sorted_by_sugar = sorted(cereals, key=lambda cereal: cereal["sugars"])
+    context.log.info(f'{sorted_by_sugar[-1]["name"]} is the sugariest cereal')
+
+
+@pipeline(mode_defs=[ModeDefinition(resource_defs={"io_manager": fs_io_manager})])
+def hello_cereal_pipeline():
+    find_sugariest(download_cereals())
+```
+
+Notice that this pipeline uses a mode with the <PyObject module="dagster" object="fs_io_manager" />. This is important because each Airflow task executes in its own process. Therefore, we need to write outputs to the filesystem so that downstream tasks can access them, instead of fetching them from memory, which is Dagster's default.
```

Comment:
the filesystem wording may confuse people who are using the remote set up. we could rephrase it to something like using persistent io managers in order to pass data between processes. and also mention s3_io_manager, gcs_io_manager for remote options.

Reply:
Good point - I'll change